packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/image.overrides

```diff
@@ -1,3 +1,38 @@
 /*******************************
     Site Overrides
 *******************************/
+
```

Comment:
Does the size should be applied to ALL images instead of scoping to avatar?

Reply:
To me an avatar should not have the same size as avatar which tend to be smaller by it's nature. Image size and avatar are something to discuss with designers.